bundles/simple/plugin-2.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"net/http"
+)
+
+// AddFooBarHeader2 adds custom "Foo: Bar" header to the request
+func AddFooBarHeader2(_ http.ResponseWriter, r *http.Request) {
+	r.Header.Add("Foo", "Bar")
+}
```

Comment:
`AddFooBarHeader` function is already in `plugin.go`. We can write a new function like below.

```suggestion
// AddHelloWorldHeader adds custom "Hello: World" header to the request
func AddHelloWorldHeader(_ http.ResponseWriter, r *http.Request) {
	r.Header.Add("Hello", "World")
}
```

Reply:
Haha ah yes I forgot to change. Have changed now :)

Reply:
Haha ah yes I forgot to change. Have changed now :)